src/aragonjs-wrapper.js

```diff
@@ -64,6 +64,25 @@ const appBaseUrl = (app, gateway = ipfsDefaultConf.gateway) => {
 const applyAppOverrides = apps =>
   apps.map(app => ({ ...app, ...(appOverrides[app.appId] || {}) }))
 
+const hasWebApp = app => Boolean(app['start_url'])
+
+const getAPMRegistry = ({ appName = '' }) =>
+  appName.substr(appName.indexOf('.') + 1) // everything after the first '.'
+
+const getAppTags = app => {
+  const apmRegistry = getAPMRegistry(app)
+  const status = app['status']
+
+  let tags = status ? [status] : []
+
+  tags =
+    apmRegistry !== 'aragonpm.eth' ? [...tags, `${apmRegistry} registry`] : tags
+
+  tags = !hasWebApp(app) ? [...tags, 'contract-only'] : tags
```

Comment:
Was wondering if we had any alternatives to "contract-only". I was thinking "No UI" but that might be confusing as well.